### ISSUE TITLE
The auxiliary loss

### ISSUE THREAD

**Issue by Ariel-JUAN** [hellochick/PSPNet-tensorflow#41]
Apart from the main branch using softmax loss to train the final classifier, another classifier is applied after the fourth stage, i.e., the res4b22 residue block. 
Do anyone know how to write the auxiliary loss code?

**Comment by gaowu9595** [comment]
I wonder, too!
